book/listings/todo_app/1/window/mod.rs

```diff
@@ -1,32 +1,32 @@
 mod imp;
 
-use std::fs::File;
-
-use crate::todo_object::{TodoData, TodoObject};
+use crate::todo_object::TodoObject;
 use crate::todo_row::TodoRow;
-use crate::utils::data_path;
 use glib::{clone, Object};
 use gtk::prelude::*;
 use gtk::subclass::prelude::*;
 use gtk::{gio, glib};
-use gtk::{Application, CustomFilter, FilterListModel, NoSelection, SignalListItemFactory};
-use log::info;
+use gtk::{Application, NoSelection, SignalListItemFactory};
```

Comment:
I personally wouldn't import this stuff either and use it via `gtk::` to make it clear where it comes from

Reply:
I don't really mind either way, but for the code snippets I want to save space.
I hope it helps that I tend to specify the namespace when I refer to it in the text